### ISSUE TITLE
Sign in with Slack: Server IP Address Could Not Be Found

### ISSUE THREAD

**Issue by KarmicXKoala** [vsls-contrib/chat#455]
After using the Sign in with Slack command and allowing the integration, I get a Chrome error that "vscode.chat’s server IP address could not be found. DNS_PROBE_FINISHED_NXDOMAIN"

Google has a cached version of vscode.chat, but I am unable to resolve the name if I try to reach it in a browser or find it with ping or nslookup

**Comment by rholland** [comment]
same problem

**Comment by pmo23** [comment]
same problem !!

**Comment by bigozzi-inspect** [comment]
Same problem!

**Comment by arjun27** [comment]
Thanks for reporting this. I'm investigating into it - for now, please use the "Chat: Configure Access Token" command to sign in with Slack.

**Comment by pmo23** [comment]
Where is the "Chat: Configure Access Token"?

**Comment by James-revans** [comment]
I generated and used a slack legacy token [https://api.slack.com/legacy/custom-integrations/legacy-tokens](url) ,and entered that as the Access Token but none of my channels or members of the workspace show up. Is there a different access token I need to generate through slack for this to work?

**Comment by KarmicXKoala** [comment]
I'm having the same problem with this as @James-revans . The legacy token made slack show up in the sidebar but does not enable any functionality at all.

**Comment by arjun27** [comment]
Thanks @James-revans and @KarmicXKoala for reporting this. I'm trying to debug, but unable to repro. My best guess is that you are facing [authorization issues](https://github.com/vsls-contrib/chat/blob/master/docs/SLACK.md#admin-approval-required) on the selected workspace

To help me debug this better, can you run the "Toggle developer tools" command in VS Code? That should give us some helpful logs.

**Comment by reidab** [comment]
Even outside of the Slack auth process, there seems to be something broken with DNS for the hosted oauth lambda service (https://github.com/vsls-contrib/chat/tree/master/oauth-service).

It claims to be set up on https://vscode.chat, but I can't find any name server that will resolve that domain. Who's controlling that domain and its configuration?

Google Public DNS:
```
➜ nslookup vscode.chat 8.8.8.8
Server:		8.8.8.8
Address:	8.8.8.8#53

** server can't find vscode.chat: SERVFAIL
```

The `.chat` name servers:
```
➜ nslookup vscode.chat DEMAND.ALPHA.ARIDNS.NET.AU
Server:		DEMAND.ALPHA.ARIDNS.NET.AU
Address:	2001:dcd:1::7#53

Non-authoritative answer:
*** Can't find vscode.chat: No answer
```

The AWS DNS provided on the whois record for `vscode.chat`:
```
➜ nslookup vscode.chat ns-1800.awsdns-33.co.uk
'Server:		ns-1800.awsdns-33.co.uk
Address:	2600:9000:5307:800::1#53

** server can't find vscode.chat: REFUSED
```

**Comment by voxpelli** [comment]
yeah `vscode.chat` seems to not be existing online anymore?

**Comment by arjunattam** [comment]
This is fixed with v0.35.0. Please use the "Sign in with Slack" command to login with Slack.

Apologies for the delay – and thank you for your patience.
